joomla/joomla-cms
# Make sure the URL installer does not allow other schemas then http and https

Pull Request for an Issue raised to the JSST.

### Summary of Changes

Make sure the URL installer does not allow other schemas then http and https

### Testing Instructions

- Try to install an extension form this url: `ftp://joomla.zip`
- Joomla tires to contact an FTP server
- apply this patch
- try `ftp://joomla.zip` again.
- There is now a dedicated message and we dont try to contact an FTP server
- make sure that extension installation from URL still works as before.

### Actual result BEFORE applying this Pull Request

There is an message but we still try to contact the FTP server

### Expected result AFTER applying this Pull Request

There is now a dedicated message and we dont try to contact an FTP server

### Documentation Changes Required

none
